src/Entity/Post.php

```diff
@@ -40,6 +40,11 @@ class Post
      */
     private $created_at;
 
+    /**
+     * @ORM\OneToMany(targetEntity=Comment::class, mappedBy="post", orphanRemoval=true)
+     */
+    private $comments;
```

Comment:
Nice job adding the orphanRemoval here. Orphaned records are a cause of many a bug